data/animations/animations.inc

```diff
@@ -992,3 +992,7 @@
 .include "data/animations/anims/anim_countdown_start.inc"
 .include "data/animations/anims/anim_1131.inc"
 .include "data/animations/anims/anim_1132.inc"
+
+.if NON_MATCHING
```

Comment:
Hmm, I'm not sure the `NON_MATCHING` flag is good for this stuff. I think we should reserve `NON_MATCHING` for when we are not replicating the original rom exactly, but the attempt is there. This is extra code.

Reply:
Yeah, I just took it because it was available.
How about ENABLE_DECOMP_CREDITS?

Reply:
Sounds fine, if we want to include these under ENHANCEMENTS or something that's also fine in the future :)